modules/migration-state-migration-cli.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/about-mtc-3-4.adoc
+// * migration_toolkit_for_containers/about-mtc.adoc
+
+[id="migration-state-migration-cli_{context}"]
+= Migrating an application's state
+
+You can use {mtc-full} ({mtc-short}) to migrate an application's state. {mtc-short} copies persistent volume claims (PVCs) and specific Kubernetes objects, for example, `Secret` custom resources (CRs) or config maps that store an application's state.
+
+If you have a CI/CD pipeline, you can migrate stateless components by deploying them on the target cluster. Then you can migrate the state of the application by selecting PVCs and Kubernetes objects.
+
+You can migrate PV data from the source cluster to PVCs that are already provisioned in the target cluster by mapping PVCs in the `MigPlan` CR. This ensures that the target PVCs of migrated applications are synchronized with the source PVCs.
+
+After you have migrated PV data, you can migrate the Kubernetes objects.
+
+[id="excluding-pvcs_{context}"]
+== Excluding persistent volume claims
```

Comment:
To follow the templates, this module should be split up into four separate modules, a concept and three separate procedures.

Reply:
But this would be pointless because none of these modules would ever be reused. So it makes more sense to keep it in a single module that will not be reused.